topology/probes/ovsdb/ovsdb.go

```diff
@@ -129,6 +127,7 @@ func (o *OvsdbProbe) OnOvsBridgeAdd(monitor *ovsdb.OvsMonitor, uuid string, row
 			}
 		}
 	}
+	o.Graph.Unlock()
```

Comment:
The graph is still used in `OnOvsBridgeAdd` so should we stay under lock. Maybe we should just Unlock and RLock

Reply:
I have put locks on "atomic" graph operations like adding a node. OnOvsBridgeAdd does much more than that and locks are not recursive in go. So I think releasing the lock is sensible but using an explicit unlock is more error prone than defer. Another solution would be to push the unlock defer rlock to the callback but relocking for an immediate unlock is weird.

Reply:
Functional tests detected a deadlock due to recursive use of graph lock. Invariants maintained here: graph is only locked on elementary actions related to it (no system stuff done while the lock is held). openflow probe lock must be locked *before* graph lock

Reply:
Race removed when we add  a group just after the creation of the bridge. Takes also into account the fact that the bridge does not recognized OF1.5 when it is created, only after a set bridge protocols=...